tests/test_zmeta.py

```diff
@@ -0,0 +1,34 @@
+from warnings import warn
+
+from helpers.constants import REPO_ROOT, S_SPY, SNAPSHOTS_USED  # type:ignore
+
+from spellbot.assets import load_strings
+
+
+# These tests will fail in isolation, you must run the full test suite for them to pass.
+# The file has a leading z in the test name to ensure it gets run last
```

Comment:
Not sure if there's a good alternative than to just make this file have a `z` before the test name

Reply:
I may try and add https://pytest-ordering.readthedocs.io/en/develop/ to do this if you haven't already by the time I get to it!

Reply:
Perfect, this worked great.